egret/common/lazy_ptdf_utils.py

```diff
@@ -484,6 +510,22 @@ def _iter_over_int_viol_set(int_viol_set, mb, PTDF, abs_ptdf_tol, rel_ptdf_tol):
             mb.pfi[i_n] = expr
         yield i, i_n
 
+def _generate_branch_thermal_bounds(mb, bn, thermal_limit, neg_slacks, pos_slacks):
+    neg_slack, pos_slack = None, None
```

Comment:
Why are `neg_slack` and `pos_slack` hard coded?

Reply:
They're not, exactly, but I've changed the logic formulation to make the intention clear.